docs/content/installation/installing-nic/installation-with-manifests.md

```diff
@@ -0,0 +1,228 @@
+
+#### Option 1: Create a NodePort service
+
+For more information about the  _NodePort_ service, refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport).
+
+1. To create a service of type *NodePort*, run:
+
+    ```shell
+    kubectl create -f service/nodeport.yaml
+    ```
+
+    Kubernetes automatically allocates two ports on every node in the cluster. You can access NGINX Ingress Controller by combining any node's IP address with these ports.
+
+#### Option 2: Create a LoadBalancer service
+
+For more information about the _LoadBalancer_ service, refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/#type-loadbalancer).
+
+1. To set up a _LoadBalancer_ service, run one of the following commands based on your cloud provider:
+
+    - GCP or Azure:
+
+        ```shell
+        kubectl apply -f service/loadbalancer.yaml
+        ```
+
+    - AWS:
+
+        ```shell
+        kubectl apply -f service/loadbalancer-aws-elb.yaml
+        ```
+
+        If yu're using AWS, Kubernetes will set up a Classic Load Balancer (ELB) in TCP mode. This load balancer will have the PROXY protocol enabled to pass along the client's IP address and port.
+
+2. AWS users: Follow these additional steps to work with ELB in TCP mode.
+
+     - Add the following keys to the `nginx-config.yaml` ConfigMap file, which you created in the [Create common resources](#create-common-resources) section.
+
+         ```yaml
+         proxy-protocol: "True"
+         real-ip-header: "proxy_protocol"
+         set-real-ip-from: "0.0.0.0/0"
+         ```
+
+     - Update the ConfigMap:
+
+         ```shell
+         kubectl apply -f common/nginx-config.yaml
+         ```
+
+    {{<note>}}AWS users have more customization options for their load balancers. These include choosing the load balancer type and configuring SSL termination. Refer to the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/service/#type-loadbalancer) to learn more. {{</note>}}
+
+3. To access NGINX Ingress Controller, get the public IP of your load balancer.
+
+    - For GCP or Azure, run:
+
+        ```shell
+        kubectl get svc nginx-ingress --namespace=nginx-ingress
+        ```
+
+    - For AWS find the DNS name:
+
+        ```shell
+        kubectl describe svc nginx-ingress --namespace=nginx-ingress
+        ```
+
+        Resolve the DNS name into an IP address using `nslookup`:
+
+        ```shell
+        nslookup <dns-name>
+        ```
+
+    You can also find more details about the public IP in the status section of an ingress resource. For more details, refer to the [Reporting Resources Status doc]({{< relref "configuration/global-configuration/reporting-resources-status.md" >}}).
+
+### Using a DaemonSet
+
+Connect to ports 80 and 443 using the IP address of any node in the cluster where NGINX Ingress Controller is running.
```

Comment:
Missing the steps for deploying as a DaemonSet

Reply:
For clarity: this revamp is largely about rewriting existing content.

https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/

There are no steps or instructions in the updated documentation beyond what already exists.

If there's missing critical paths it would be preferable if they were filed as their own issues to action.